src/screens/scanning/VoucherEntryNavigator.tsx

```diff
@@ -6,124 +6,96 @@ import {
   TouchableOpacity,
   KeyboardAvoidingView,
   Platform,
-  View,
 } from 'react-native';
 import Toast from 'react-native-toast-message';
-import { SafeArea } from '../../../assets/Components';
-import { NavButtonText, TitleText } from '../../../assets/Fonts';
+import { Row, SafeArea } from '../../../assets/Components';
+import { H2Heading } from '../../../assets/Fonts';
 import StandardHeader from '../../components/common/StandardHeader';
 import { VoucherEntryNavigationProps } from '../../navigation/types';
 import Colors from '../../../assets/Colors';
 import { useScanningContext } from './ScanningContext';
 import { handlePreventLeave } from '../../utils/scanningUtils';
-import styles, { NavButton, NavButtonContainer } from './styles';
 import ManualVoucherScreen from './ManualVoucherScreen';
 import ScanningScreen from './ScanningScreen';
 import VoucherBatchScreen from './VoucherBatchScreen';
+import NavButton from '../../components/scanning/NavButton';
 
-interface VoucherEntryNavigatorProps {
+type VoucherEntryNavigatorProps = {
   navigation: VoucherEntryNavigationProps;
+};
+
+enum tabs {
+  SERIAL,
+  RANGE,
+  BARCODE,
 }
 
 export default function VoucherEntryNavigator({
   navigation,
 }: VoucherEntryNavigatorProps) {
-  const [selection, setSelection] = useState(1);
+  const [selection, setSelection] = useState(tabs.SERIAL);
   const { voucherMap, dispatch } = useScanningContext();
   const hasUnsavedChanges = Boolean(voucherMap.size);
 
+  const onPreventLeave = () =>
+    handlePreventLeave({
+      hasUnsavedChanges,
+      navigation,
+      dispatch,
+    });
+
+  const onSelectSerial = () => setSelection(tabs.SERIAL);
+
+  const onSelectRange = () => setSelection(tabs.RANGE);
+
+  const onSelectBarcode = () => setSelection(tabs.BARCODE);
+
+  const handleNavigate = () => {
+    switch (selection) {
+      case tabs.SERIAL:
+        return <ManualVoucherScreen navigation={navigation} />;
+      case tabs.RANGE:
+        return <VoucherBatchScreen navigation={navigation} />;
+      case tabs.BARCODE:
+        return <ScanningScreen navigation={navigation} />;
+      default:
+        return <ManualVoucherScreen navigation={navigation} />;
+    }
+  };
+
   return (
     // this line enables the keyboard to be dismissed when the user taps anywhere on the screen
     <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
       <SafeArea>
-        <StandardHeader width="80%">
-          <TitleText>Add a voucher</TitleText>
-          <TouchableOpacity
-            onPress={() =>
-              handlePreventLeave({
-                hasUnsavedChanges,
-                navigation,
-                dispatch,
-              })
-            }
-          >
+        <StandardHeader>
+          <H2Heading>Add a voucher</H2Heading>
+          <TouchableOpacity onPress={onPreventLeave}>
```

Comment:
Don't know if this is an issue introduced in this pr, but there isn't much spacing above the top bar header. Especially with the notch on iPhones 10-13, the header rubs against the top of the screen.
![IMG_EE17B4F39994-1](https://github.com/calblueprint/vouchers4veggies/assets/66931067/cb093ced-430e-4526-a806-3783d72cf28b)

Reply:
I don't think I changed the header spacing so the only thing I can think of would be that the SafeAreaView is no longer working as intended. If you replace the `SafeArea` with a `SafeAreaView` imported from react native, does this solve the issue?